package.json

```diff
@@ -53,39 +52,40 @@
     "@types/cookie-parser": "^1.4.2",
     "@types/express": "^4.17.13",
     "@types/express-session": "^1.17.4",
-    "@types/jest": "^27.0.3",
+    "@types/jest": "^27.4.0",
     "@types/node": "^16.11.10",
-    "@types/superagent": "^4.1.13",
-    "@typescript-eslint/eslint-plugin": "^5.4.0",
-    "@typescript-eslint/parser": "^5.4.0",
+    "@types/superagent": "^4.1.15",
+    "@typescript-eslint/eslint-plugin": "^5.10.2",
+    "@typescript-eslint/parser": "^5.10.2",
     "audit-ci": "^5.1.2",
-    "concurrently": "^6.4.0",
-    "cypress": "^9.1.0",
+    "concurrently": "^7.0.0",
+    "cypress": "^9.4.1",
     "cypress-multi-reporters": "^1.5.0",
-    "dotenv-expand": "^5.1.0",
-    "eslint": "^8.3.0",
+    "dotenv-expand": "^6.0.1",
+    "eslint": "^8.8.0",
     "eslint-config-airbnb-base": "^15.0.0",
     "eslint-config-prettier": "^8.3.0",
     "eslint-import-resolver-typescript": "^2.5.0",
     "eslint-plugin-cypress": "^2.12.1",
-    "eslint-plugin-import": "^2.25.3",
+    "eslint-plugin-import": "^2.25.4",
     "eslint-plugin-no-only-tests": "^2.6.0",
     "eslint-plugin-prettier": "^4.0.0",
     "husky": "^7.0.4",
-    "jest": "^27.3.1",
-    "jest-html-reporter": "^3.4.1",
+    "jest": "^27.4.7",
+    "jest-html-reporter": "^3.4.2",
     "jest-junit": "^13.0.0",
-    "jsdom": "^18.1.1",
-    "lint-staged": "^12.1.2",
+    "jsdom": "^19.0.0",
+    "lint-staged": "^12.3.3",
     "mocha-junit-reporter": "^2.0.2",
-    "node-sass": "^6.0.1",
+    "nock": "^13.2.2",
+    "node-sass": "^7.0.1",
     "nodemon": "^2.0.15",
     "nyc": "15.1.0",
-    "prettier": "2.4.1",
+    "prettier": "2.5.1",
     "set-cookie-parser": "^2.4.8",
-    "supertest": "^6.1.6",
-    "ts-jest": "^27.0.7",
-    "typescript": "^4.5.2"
+    "supertest": "^6.2.2",
+    "ts-jest": "^27.1.3",
+    "typescript": "^4.5.5"
```

Comment:
there's some known issues with:
* redis
* MOJ frontend
* Govuk frontend


We'll need to create a new ticket for dealing with these